examples/external_ip/main.tf

```diff
@@ -16,7 +16,7 @@
 
 provider "google" {
   credentials = file(var.credentials_path)
-  version     = "~> 2.11.0"
+  version     = "~> 3.7.0"
```

Comment:
Instead of `~> 3.7.0` can we make this `~> 3.7`? This will need to be made in several other places.

Reply:
Done.

Reply:
Just to make sure I understand the rationale behind it correctly. Putting 3.7 here instead of 3.7.0 is to leave it open so any version >= 3.7.0 and < 4.0 is supported. According to the documentation regarding ~> here https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version

Reply:
Correct. For examples, I usually pin to a minor version (ie. `~> 3.7.0`) while the module itself is less restrictive but either is acceptable.

Reply:
Thanks.